src/replayToMovie.js

```diff
@@ -55,13 +56,18 @@ require('@code-dot-org/p5.play/lib/p5.play');
 
 const p5Inst = new P5(function (p5obj) {
   p5obj._fixedSpriteAnimationFrameSizes = true;
+  p5obj.width = WIDTH;
+  p5obj.height = HEIGHT;
 });
```

Comment:
Can we get updated to use `loadP5.js`?

Reply:
unfortunately not without significant changes; I explored that, but we're using two different versions of Canvas in this and in the dance-party repo, with two different sets of requirements for the workaround.

I'd definitely like to explore unifying them at some point, but likely after HoC